pkg/constant/fast_confirmation_multiplier.go

```diff
@@ -0,0 +1,72 @@
+package constant
+
+import (
+	"math/big"
+
+	sdkmath "cosmossdk.io/math"
+
+	"github.com/zeta-chain/node/pkg/chains"
+)
+
+var (
+	// DefaultInboundFastConfirmationLiquidityMultiplier is the default ZRC20 liquidity cap multiplier for inbound fast confirmation
+	// The reason of using decimal type is for easy integration into ChainParams, like field `ballot_threshold`
+	DefaultInboundFastConfirmationLiquidityMultiplier = sdkmath.LegacyMustNewDecFromStr("0.00025")
+
+	// InboundFastConfirmationLiquidityMultiplierMap maps chainID to ZRC20 liquidity cap multiplier for inbound fast confirmation.
+	// Fast inbound confirmation is enabled only for chains explicitly listed in this map.
+	InboundFastConfirmationLiquidityMultiplierMap = map[int64]sdkmath.LegacyDec{
```

Comment:
Why do we need this map if DefaultInboundFastConfirmationLiquidityMultiplier is a default value? This makes more code to maintain when adding new chains

Reply:
It is a good question similar to [first comment](https://github.com/zeta-chain/node/pull/3551#discussion_r1962054347) by @CharlieMc0

Reply:
same refactor https://github.com/zeta-chain/node/pull/3551/commits/6596863f4b5045e05799cfd06e5e40543ca2c90f